test-network-nano-bash/configtx.yaml

```diff
@@ -215,52 +215,6 @@ Application: &ApplicationDefaults
 ################################################################################
 Orderer: &OrdererDefaults
 
-    # Orderer Type: The orderer implementation to start
-    OrdererType: etcdraft
```

Comment:
This was removed in the PR: test network nano bash updated with smartbft consensus #923
upon I added the channel-participation.

Reply:
As a pull request author you are also responsible for the previous commits, so I'm asking you why in your opinion it is needed to remove this.

Reply:
But since you can override the orderer type in the specific profile, why not leave Raft as default like earlier and just override it when you need to use BFT?

Reply:
I guess you can do that, from my perspective it's a matter of design.
I think that specifying the orderer type explicitly gives some  awareness to the user that tries to configure the network. If you take a user that wants to run something simple, then a some profile doesn't work - he could see right away what type of orderer he's using.
But it's up to you.

Reply:
Let's keep it as it is now. No need to revert.